doc/examples/voidlinux-musl

```diff
@@ -0,0 +1,82 @@
+image:
+  distribution: voidlinux
+  description: Void Linux
+  expiry: 30d
+  architecture: x86_64
+
+source:
+  downloader: voidlinux-http
+  url: https://alpha.de.repo.voidlinux.org/live/current
```

Comment:
I defaulted this to https because otherwise I get the same issue as has already been reported in #197

Reply:
Same as above, hopefully this works now